crates/libs/core/src/strings/hstring.rs

```diff
@@ -19,7 +19,7 @@ impl HSTRING {
         self.0.is_none()
     }
 
-    /// Returns the length of the string.
+    /// Returns the length of the string. The length is measured in `u16`s, not including the terminating null character.
```

Comment:
I think it's important to additionally indicate these are UTF-16 code units, otherwise looks good to me.


```suggestion
    /// Returns the length of the string. The length is measured in `u16`s (UTF-16 code units), not including the terminating null character.
```